packages/carbon-graphs/src/js/controls/Gantt/helpers/datapointHelpers.js

```diff
@@ -30,6 +30,7 @@ export const drawDataPoints = (
   dataTarget,
   drawDataPointsHandler,
   event,
+  legendSVG,
```

Comment:
nit: JS description not updated with the new parameter `legendSVG`

Reply:
updated here https://github.com/cerner/terra-graphs/pull/18/commits/30019d1a1afbecf20ff1b8476b29e225df1cc68b